pootle/checks.py

```diff
@@ -490,3 +490,36 @@ def check_canonical_url(app_configs=None, **kwargs):
     elif settings.POOTLE_CANONICAL_URL == "http://localhost":
         errors.append(localhost_canonical_warning)
     return errors
+
+
+@checks.register()
+def check_pootle_fs_working_dir(app_configs=None, **kwargs):
+    import os
+
+    from django.conf import settings
+
+    missing_setting_error = checks.Critical(
+        _("POOTLE_FS_WORKING_PATH setting is not set."),
+        hint=_("Set `POOTLE_FS_WORKING_PATH` in settings"),
+        id="pootle.C019",
+    )
+    missing_directory_error = checks.Critical(
+        _("Path pointed by POOTLE_FS_WORKING_PATH doesn't exist."),
+        hint=_("Create the directory pointed by `POOTLE_FS_WORKING_PATH` "
+               "setting."),
+        id="pootle.C020",
+    )
+    not_writable_directory_error = checks.Critical(
+        _("Path pointed by POOTLE_FS_WORKING_PATH is not writable by Pootle."),
+        hint=_("Add write permission for the directory pointed by "
+               "`POOTLE_FS_WORKING_PATH` setting."),
```

Comment:
Add the write permissions to the `POOTLE_FS_WORKNG_PATH` directory using 'chmod +w'.

Hints are not meant to just repeat the error.